web/src/components/tables/AccessRuleTable.tsx

```diff
@@ -132,7 +132,11 @@ export const AccessRuleTable = () => {
             <HStack>
               <UserAvatarDetails
                 tooltip
-                user={cell.value?.createdBy}
+                user={
+                  cell.value?.createdBy !== ""
+                    ? cell.value?.createdBy
+                    : "Common Fate Terraform Bot"
```

Comment:
Wasn't a massive fan of doing this, it would otherwise show up blank in the frontend. Keen to hear some feedback on this

Reply:
@meyerjrr the "created by" field in the backend should be attributed to the Common Fate Terraform Bot, rather than just stored in the frontend IMO

Reply:
@meyerjrr isn't there already handling in the `UserAvatarDetails` component for this case?

My concern here by the way is that in future we might add additional types of service accounts - for example, a rule could be created by `bot_python_sdk`. This code here could cause bugs in future where "Common Fate Terraform Bot" is incorrectly rendered